test/e2e/serial/tests/non_regression.go

```diff
@@ -116,7 +116,7 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			numOfNodeToBePadded := len(nrts) - 1
 
 			rl := corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("1"),
+				corev1.ResourceCPU:    resource.MustParse("2"),
 				corev1.ResourceMemory: resource.MustParse("4G"),
```

Comment:
TAE means we don't have enough resources, as confirmed by the commit message. So we fix the lack of resources by... requiring more resources?

Reply:
these are resources to leave after the padding on each numazone, so this is in the setup, my bad on not elaborating this in the commit and clarify this and recommit

Reply:
right right, scratch my comment then.